<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QMainWindow" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#SettingWindow{border-image:url(:/images/start_bgi.png);}
QLabel{color:white;}
QGroupBox{color:white}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="music">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#music{background-color:rgba(0, 0, 0, 80);}</string>
    </property>
    <property name="title">
     <string>音乐设定</string>
    </property>
    <widget class="QLabel" name="musicLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏时音乐：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="musicPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="musicBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/images/gameIcon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black
</string>
    </property>
    <property name="text">
     <string>五子棋</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="image">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#image{background-color:rgba(0, 0, 0, 80);}</string>
    </property>
    <property name="title">
     <string>图像设定</string>
    </property>
    <widget class="QLineEdit" name="bgiPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bgiLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏时背景：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bgiBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="whitePath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="whiteLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>白棋样式：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="whiteBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="blackPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="blackLbl2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>黑棋样式：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blackBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="game">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#game{background-color:rgba(0, 0, 0, 80);}</string>
    </property>
    <property name="title">
     <string>游戏设定</string>
    </property>
    <widget class="QLabel" name="undoTimesLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>悔棋次数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="whiteLbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏倒计时：</string>
     </property>
    </widget>
    <widget class="QLabel" name="AIDelayLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AI延迟时间：</string>
     </property>
    </widget>
    <widget class="QSlider" name="undoTimes">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="countTime">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="AIDelay">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="undoTimesEnter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="countTimeEnter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AIDelayEnter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="BlueButton" name="defaultBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="BlueButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <zorder>iconLabel</zorder>
   <zorder>music</zorder>
   <zorder>titleLabel</zorder>
   <zorder>image</zorder>
   <zorder>game</zorder>
   <zorder>defaultBtn</zorder>
   <zorder>returnBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlueButton</class>
   <extends>QPushButton</extends>
   <header>bluebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
